config/origin_defaults.py

```diff
@@ -1,2 +1,5 @@
+ORIGIN_SIGNING_KEY = ("0x1fc2b755568ce8402e422f8fd0da54d3"
+                      "84f42962c8f925116964f39245d429e0")
```

Comment:
We don't need `ORIGIN_SIGNING_KEY` any more. We renamed to `ATTESTATION_SIGNING_KEY`.